include/astshim/Mapping.h

```diff
@@ -105,16 +104,36 @@ friend class Object;
     bool getReport() const { return getB("Report"); }
 
     /**
-    Get @ref Mapping_TranForward "TranForward": is the forward transform defined?
+    Is the forward transform available?
+
+    Note: this gets the @ref Mapping_TranForward "TranForward" attribute,
```

Comment:
I guess we could ask @jonathansick to make it less prominent in the `breathe` output. I think the styling is still TBD.